src/Servers/IIS/AspNetCoreModuleV2/CommonLib/config_utility.h

```diff
@@ -40,6 +41,13 @@ class ConfigUtility
         return FindKeyValuePair(pElement, CS_ASPNETCORE_DEBUG_LEVEL, strDebugFile);
     }
 
+    static
+    HRESULT
+    FindDisableRedirectionOutOfProcess(IAppHostElement* pElement, STRU& strForwardResponseConnectionHeader)
+    {
+        return FindKeyValuePair(pElement, CS_ASPNETCORE_DISABLE_REDIRECTION_OUT_OF_PROCESS, strForwardResponseConnectionHeader);
```

Comment:
Redirection is an overloaded term in the server. It needs to say what is being redirected.
Would be nice to have an abreviated convention for marking in-proc vs out-of-proc. E.g. out_disableConsoleRedirection or oop_disableConsoleRedirection.

Reply:
@anurse thoughts on the negative flag name?

Reply:
Yeah, we should avoid negative names. Can it be called `EnableConsoleRedirection` and just default to `true` if not specified?

Reply:
`EnabledOutOfProcessRedirection`? (since this is not specific to console and specific to out of process)

Reply:
Settled with EnabledOutOfProcessConsoleRedirection

Reply:
`Enable` though, not `Enabled`, right? :)

Reply:
I read the code, it's good ;)